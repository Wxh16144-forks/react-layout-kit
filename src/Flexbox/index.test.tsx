import { render } from '@testing-library/react';

import { Flexbox } from 'react-layout-kit';

describe('Flexbox', () => {
  it('gap', () => {
    const { container } = render(
      <Flexbox horizontal gap={2}>
        <div>1</div>
        <div>2</div>
      </Flexbox>,
    );

    expect(container).toMatchSnapshot();
  });
});
