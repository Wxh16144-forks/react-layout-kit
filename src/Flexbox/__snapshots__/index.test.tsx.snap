// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flexbox gap 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c0 > *:not(:last-child) {
  margin-right: 2px;
}

<div>
  <div
    class="c0 layoutkit-flexbox"
  >
    <div>
      1
    </div>
    <div>
      2
    </div>
  </div>
</div>
`;
